content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -346,20 +343,23 @@ and [external.metrics.k8s.io](https://github.com/kubernetes/community/blob/maste
 For examples of how to use them see [the walkthrough for using custom metrics](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics)
 and [the walkthrough for using external metrics](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-metrics-not-related-to-kubernetes-objects).
 
-## Support for configurable scaling behavior
+## Configurable scaling behavior
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
 
-Starting from
-[v1.18](https://github.com/kubernetes/enhancements/blob/master/keps/sig-autoscaling/853-configurable-hpa-scale-velocity/README.md)
-the `v2beta2` API (and from v1.23 the `v2` API) allows scaling
-behavior to be configured through the HPA `behavior` field. Behaviors
-are specified separately for scaling up and down in `scaleUp` or
-`scaleDown` section under the `behavior` field. A stabilization window
-can be specified for both directions which prevents the flapping of
-the number of the replicas in the scaling target. Similarly specifying
-scaling policies controls the rate of change of replicas while
-scaling.
+(the `autoscaling/v2beta2` API version previously provided this ability as a beta feature)
 
-### Scaling Policies
+If you use the `v2` HorizontalPodAutoscaler API, you can use the `behavior` field
+(see the [API reference](/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2/#HorizontalPodAutoscalerSpec))
+to configure separate scale-up and scale-down behaviors.
+You specify these behaviours by setting `scaleUp` and / or `scaleDown`
+under the `behavior` field.
+
+You can specify a _stabilization window_ that prevents [flapping](#flapping)
```

Comment:
Maybe `mitigates` or `lessens` is better here, as we can't guarantee preventing any scaling flapping.

Reply:
Actually, do we need this segment given largely restating it ~40 lines below?